raiden/network/proxies/token_network.py

```diff
@@ -898,6 +898,12 @@ def close(
     ):
         """ Close the channel using the provided balance proof.
 
+        Note:
+            This method must *not* be called without updating the application
```

Comment:
By application state you mean the `channel_state` (open/closed/settled) of `NettingChannelState`?

Reply:
no, I mean Raiden itself

Reply:
I don't fully understand the second part I guess, which leads to my confusion:

> otherwise the node may accept new transfers which cannot be used, because the closer is not allowed to update the balance proof sent.

So you mean that since we close the channel we must make sure alter the raiden state to stop accepting any new transactions, because if we by accident accept any we won't be able to update the onchain state. That's because the closer get only one chance to update what he received and that's only at closing?

Reply:
yes

Reply:
Okay good. Then the one suggestion I have to make it clearer is to change:

`because the closer is not allowed to update the balance proof sent` -> `because the closer is not allowed to update the balance proof submitted on chain after closing`.

Reply:
done